doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,353 @@
+# <a name="top"></a>List of HTTP Headers used by Orion
+
+1)  [Accept](#1-accept--)        
+2)  [Content-Length](#2-content-length-)         
+3)  [Content-Type](#3-content-type-)
+4)  [Connection]()                
+5)  [Expect](#5-expect-)                                                                                
+6)  [Host](#6-host)                        	                                                      
+7)  [Origin](#7-origin)                                                       
+8)  [User-Agent](#8-user-agent-)              
+9)  [X-Forwarded-For](#9-x-forwarded-for)                                                               
+10) [X-Real-IP](#10-x-real-ip-)
+
+11) [Access-Control-Allow-Origin](#11--access-control-allow-origin-)
+12) [Access-Control-Allow-Headers](#12-access-control-allow-headers-)                                                                   
+13) [Access-Control-Allow-Methods](#13-access-control-allow-methods-)
+14) [Access-Control-Max-Age](#14-access-control-max-age-)                                 
+15) [Access-Control-Expose-Headers](#15-access-control-expose-headers-)
+16) [Allow](#16-allow-)                                                                                    
+17) [Location](#17-location-)                                                 
+18) [Ngsiv2-AttrsFormat](#18-ngsiv2-attrsformat-)                                    
+19) [Fiware-Service](#19-fiware-service-)
+20) [Fiware-Servicepath](#20-fiware-servicepath-)
+21) [Fiware-Total-Count](#21-fiware-total-count-)
+22) [Fiware-Correlator](#22-fiware-correlator-)                                     
+
+23) [X-Auth-Token](#23-x-auth-token)
+
+
+
+# HTTP Headers used by Orion with their Description:
+
+
+
+## 1. Accept  : 
```

Comment:
```suggestion
## 1. Accept
```

Let's avoid trailing ":" in title (to use the same style that other parts in the documentation)

Reply:
fixed in 469e9c3